packages/libllvm/build.sh

```diff
@@ -88,6 +92,9 @@ termux_step_pre_configure() {
 	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DLLVM_HOST_TRIPLE=$LLVM_DEFAULT_TARGET_TRIPLE"
 	export TERMUX_SRCDIR_SAVE=$TERMUX_PKG_SRCDIR
 	TERMUX_PKG_SRCDIR=$TERMUX_PKG_SRCDIR/llvm
+
+	sed -i "s|$<TARGET_FILE:mlir-tblgen>|$TERMUX_PKG_HOSTBUILD_DIR/bin/mlir-tblgen|" $TERMUX_PKG_SRCDIR/../flang/CMakeLists.txt
```

Comment:
Is  `$TERMUX_PKG_SRCDIR/../flang` correct here? Do you have the folder `~/.termux-build/libllvm/flang?

Reply:
it's ~/.termux-build/libllvm/src/flang. see line 94

Reply:
oh, sorry, did not see the previous line there. Works as it is now then